packages/component/src/Styles/createStyleSet.js

```diff
@@ -68,7 +68,14 @@ export default function createStyleSet(options) {
   // Keep this list flat (no nested style) and serializable (no functions)
 
   // TODO: [P4] Deprecate this code after bump to v5
```

Comment:
file an issue?

Reply:
Let's do it as a separate pass. We have lots of "deprecate this after MM-DD-YY" in our codebase.